packages/e2e/local-network/scripts/pools/update-node-utils.sh

```diff
@@ -129,7 +129,7 @@ updatePool() {
     --protocol-params-file ${SP_NODE_ID}/params.json | awk '{ print $1 }')
 
   initialBalance=$(getAddressBalance "$genesisAddr")
-  txOut=$((initialBalance - fee))
+  txOut=$((initialBalance - fee - 2000000))
```

Comment:
Is this for keyDeposit that was previously 0?
```suggestion
  txOut=$((initialBalance - fee - keyDeposit))
```

Reply:
Yes. 001e4043df should solve this